docs/resources/getting_started/client-ddsrouter.yaml

```diff
@@ -3,7 +3,6 @@
 allowlist:
   - name: Square
     type: ShapeType
```

Comment:
I would remove type as well. The topic is known by the user (it decides the figure shape), but it does not know the type.